Mac/Browser.swift

```diff
@@ -73,3 +73,42 @@ extension Browser {
 			NSLocalizedString("Open in Browser in Background", comment: "Open in Browser in Background menu item title")
 	}
 }
+
+extension Browser {
+
+	static func open(_ urlStrings: [String], alertingInWindow window: NSWindow?, invertPreference: Bool = false) {
```

Comment:
This is to avoid lengthy code duplication.

Considered alternatives:

- Having this method on the `MainWindowController`. This would have the benefit of not needing the `window` parameter, but would require some sort-of-ugly `as? MainWindowController` stuff in the timeline view controller.
- Creating a `BrowserURLOpenable` protocol (with a better name), that provided this default implementation and required a `window` property (which `MainWindowController` already has). However, this `open` method should never be overridden.

One thing that could be changed is doing the check for the shift key in this method rather than passing in `invertPreferences`, since we're trying to unify behavior anyway.